home/tests/test_api.py

```diff
@@ -947,7 +951,9 @@ def test_list_items(self, uclient):
         """
         test that list items are present in the whatsapp message
         """
-        page = self.create_content_page(list_items=["list item 1", "list item 2"])
+        page = self.create_content_page(
+            list_button_title="List Button", list_items=["list item 1", "list item 2"]
+        )
```

Comment:
Do we have a test for a list with no title?

Reply:
Added one